lib/winapi/sysinfo.c

```diff
@@ -2,6 +2,28 @@
 #include <assert.h>
 #include <xboxkrnl/xboxkrnl.h>
 
+void GetSystemTime (LPSYSTEMTIME lpSystemTime)
+{
+    assert(lpSystemTime != NULL);
+
+    LARGE_INTEGER KTime;
+    TIME_FIELDS TimeFields;
+
+    KeQuerySystemTime(&KTime);
+
```

Comment:
nit: I feel like this blank line doesn't serve any purpose.

Reply:
Removed blank line 13 to make a bloc after declarations